demos/launch/dispatcher.launch.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' ?>
```

Comment:
I recommend copying the arguments and the code block under "Dispatcher GUI Server"  into `common.launch.xml` under this block https://github.com/osrf/rmf_demos/blob/7bce0433f67eae0719324a5fe8c63a905dce556b/demos/launch/common.launch.xml#L46-L52

We can then delete this file

Reply:
hi Yadu, I think I misinterpreted our discussion in the office about the configuration json file previously, but I've clarified with @tanyouliang95 this afternoon

how rmf-web does it is through this [script ](https://github.com/osrf/rmf-web/blob/main/packages/dashboard/scripts/setup/setup.js) that the user has to run with `npm run setup` in the terminal before being able to launch the dashboard successfully. it provides a path to the resources needed for the dashboard to work (currently it allows retrieving the resources from a remote github repo or using an absolute path set by the user to retrieve the resources by creating a `.resources.json` file
![image](https://user-images.githubusercontent.com/28668944/104014554-2cc36000-51ee-11eb-890e-e7b11bb14696.png)
resulting `.resources.json` file
`{"repoUrl":"https:github.com/osrf/rmf_demos.git","folder":"rmf_dashboard_resources/office/","branch":"master"}`

the current implementation for the rmf  panel assumes that that config file is available locally in the `static` folder, hence when the user navigates between the tabs, the form options are updated accordingly. 

I'm not sure if there can be relative paths using the way that rmf-web does it, I could look more into it, but in that case, do I assume that we want the config to be retrieved locally? instead of being hosted on a server and then the config is fetched with a HTTP request or smth like that?

On your last point, I've made the changes to the titles :+1:

Reply:
https://github.com/osrf/rmf_demos/pull/178. `common.launch.xml` will include `dashboard.launch.xml` and `dispactcher_node`.